### PR TITLE
Add homepage, author, description to about:addons

### LINKED ISSUE #244: Price Wise experiment doesn't have the author, description and homepage sections in "about:addons" page
[Affected versions]:
- Firefox Release v63.0 and up

[Affected Platforms]:
- All Windows
- All Linux
- All Mac

[Prerequisites]:
- Have a Firefox profile with the latest version of the "Price Wise" add-on (8.0.1) downloaded from ([download link](https://usercontent.irccloud-cdn.com/file/LCrGGoJF/price_wise-8.0.1-signed.xpi)). 

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Navigate to "about:addons" page, Extensions section.
3. Click on the Price Wise experiment.
4. Observe the author, description and homepage areas.

[Expected result]:
- The author, a brief description, and the homepage of the experiment are displayed on the experiment's page.

[Actual result]:
- The author, a brief description, and the homepage of the experiment sections are not displayed at all.

[Notes]:
- Here is a screenshot of the issue compared to other Test Pilot experiments:

about addons

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #244

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "price-wise",
   "version": "8.0.0",
-  "description": "",
-  "private": "true",
+  "description": "Price Wise is a Firefox extension that tracks price changes to help you find the best time to buy.",
```
[the remark this answers is not in the document]
Comment (chuckharmston) [reply package.json]: ☑️

File: src/manifest.json [hunk 2]
```diff
@@ -2,7 +2,9 @@
   "manifest_version": 2,
   "name": "Price Wise",
   "version": "$version",
-  "description": "",
+  "author": "$author",
```
[the remark this answers is not in the document]
Comment (chuckharmston) [reply src/manifest.json]: Yeah, `Mozilla` would be correct here.

File: package.json [hunk 3]
```diff
@@ -62,5 +62,14 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/price-wise.git"
+  },
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/mozilla/price-wise/issues"
+  },
+  "homepage": "https://testpilot.firefox.com/experiments/price-wise"
```
[the remark this answers is not in the document]
Comment (chuckharmston) [reply package.json]: Test Pilot experiments have historically done that, yeah:

File: package.json [hunk 4]
```diff
@@ -62,5 +62,14 @@
     "redux": "4.0.0",
     "redux-thunk": "2.3.0",
     "uuid": "3.3.2"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/price-wise.git"
```
[the remark this answers is not in the document]
Comment (pdehaan) [reply package.json]: No good reason, that was just the `npm init` default format.

**Comment by biancadanforth** [comment]
Thanks @pdehaan !

**Pull request merged**